Sources/App/Views/PackageController/Builds/BuildIndex+View.swift

```diff
@@ -27,7 +27,7 @@ enum BuildIndex {
         }
 
         override func pageTitle() -> String? {
-            "\(model.packageName) &ndash; Build Results"
+            "\(model.packageName ?? model.repositoryName) &ndash; Build Results"
```

Comment:
It's obviously not ideal that we now leak logic for dealing with nil package names into the view. However, it also didn't feel right to populate a field named `packageName` with a repository name.

Reply:
I think that's what you mean here, don't you? The reason I'd say that's ok is because `Model` isn't a db model but a transition view model where we prep the data for the view. We could rename the field but I don't think there's a better name, nor should it be confusing that the view's package name can fall back to the owner.